tap_pendo/discover.py

```diff
@@ -56,6 +56,10 @@ def get_schema_property_type(schema_type):
         return {"type": ["null", "string"], "format": "date-time"}
     elif schema_type == 'boolean':
         return {"type": ["null", "boolean"]}
+    elif schema_type == 'integer':
+        return {"type": ["null", "integer"]}
+    elif schema_type == '':
+        return {}
```

Comment:
Maybe personal preference, but I like `dict()` over `{}` to remove ambiguity between whether its a set or dict.

Reply:
I approve with or without this change, unfortunately I can't approve my own pr ☹️